web/main.js

```diff
@@ -136,13 +138,9 @@ let receive = e => {
 	}
 	if (receiving.offset == receiving.data.length) {
 		let blob = new Blob([receiving.data])
-		let a = document.createElement('a');
-		a.href = URL.createObjectURL(blob); // TODO release this?
-		a.download = receiving.name;
-		a.style.display = 'none';
-		document.body.appendChild(a);
-		a.click();
-		document.body.removeChild(a);
+		receiving.a.href = URL.createObjectURL(blob); // TODO release this?
```

Comment:
you can get rid of the `// TODO release this?` :) if we're keeping it as a link we don't want to release it.